docs/how-to-build.md

```diff
@@ -80,6 +80,15 @@ This software is licensed under the Apache License version 2."
     ```shell
     ./gradlew build -PpythonVersion=3.11
     ```
+   
+4. Build Gravitino Spark connector
+
+    ```shell
+    ./gradlew spark-connector:spark-connector-runtime:shadowJar
```

Comment:
I think normal `build` should be enough, can you please confirm that?

Reply:
yes,  it works,  the dependency chain is shadowJar -> Jar -> build. updated the doc

Reply:
So please update the doc to use `build`, not `shadowJar`.

Reply:
updated, please check it again